ansible/install-apps.yml

```
@@ -0,0 +1,29 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
```

I believe it shouldn't be a sequence of mappings, just a mapping
```suggestion
    apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
```